template/Dockerfile/images/nginx.jinja2

```diff
@@ -14,11 +14,14 @@
         nginx
 {%- endmacro %}
 
-
-{% macro debian(role='') -%}
+{% macro debian(role='',osCodename='buster') -%}
     # Install nginx
+    && printf "deb http://nginx.org/packages/debian/ {{ osCodename }}  nginx\n deb-src http://nginx.org/packages/debian/ {{ osCodename }} nginx" \
```

Comment:
You don't need to manually write the codename. You can do `$(docker-image-info dist-codename)`. This way you don't need to introduce new arguments to the macro.

Reply:
Will change that !